src/main/java/com/github/timm/cucumber/generate/GenerateRunnersMojo.java

```diff
@@ -190,13 +190,20 @@
     public void execute() throws MojoExecutionException {
 
         if (!featuresDirectory.exists()) {
-            throw new MojoExecutionException("Features directory does not exist");
+            getLog().warn("Features directory does not exist");
+            return;
         }
 
+        createOutputDirIfRequired();
+
         final Collection<File> featureFiles = listFiles(featuresDirectory, new String[] {"feature"}, true);
-        final List<File> sortedFeatureFiles = new NameFileComparator().sort(new ArrayList<File>(featureFiles));
 
-        createOutputDirIfRequired();
+        if (featureFiles.isEmpty()) {
+            getLog().warn("Features directory is empty. No runners will be generated.");
```

Comment:
Surefire informs us that there were no tests to run. 
```
[INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ simmetrics-example ---
[INFO] No tests to run.
```

I think this should be `No runners to generate`. 

It is also possible that when there are feature files, none generate runners due to a combination of tags.  So I think this notification should be moved to the `CucumberITGenerator` implementations.